webdistill.css

```diff
@@ -1,4 +1,41 @@
-/*@media screen and (min-width: 768px)*/
+@media screen and (max-width: 900px) {

```

Comment:
Because of how CSS inheritance works, I recommend moving all your `@media` queries to the bottom of your file. That was all your default (mobile-first) styles are applied.